### DISCUSSION TITLE
Is it possible/desirable to combine e.g. `DP-x` and `desc:Monitorname` to achieve monitor config selection? // Force monitor override with `hyprctl`?

### DISCUSSION THREAD

**Discussion by ppenguin** [hyprwm/Hyprland#2270] [Q&A] [answered]
Back in the "`X/autorandr`" days, i'd plugin my external monitor via my USB-C-docking-dongle in one USB port to get it positioned left (detected as DP-1), and in the other port (detected as DP-2) to get it positioned right to `eDP-1`.

To make this work from the config, I guess it should be possible to specify both the `DPx` part and the `desc:` part for the `monitor` keyword?

Additionally, if I execute `hyprland keyword monitor` statements on a monitor that is defined in the config by `desc:Dell Inc....`, it doesn't appear to work, possibly because the def in the config is considered "most specific" and takes priority?
Would it be an idea to add a "force" override functionality to `hyprctl` in some way for this?

Or for that matter, we could consider that if a user executed `hyprctl`, his intention is probably always to *override* a static config, so it should always take priority?

**Comment by vaxerski** [comment, marked as answer]
no it's not, feel free to open a feature req
